index.d.ts

```diff
@@ -304,16 +304,29 @@ declare global {
 
     export type Paint = SolidPaint | GradientPaint | ImagePaint
 
-    export interface ShadowEffect {
-      type: 'inner-shadow' | 'drop-shadow'
+    export interface DropShadowEffect {
+      type: 'drop-shadow'
       color: HexCode | Color
       offset: Vector
       blur: number
       blendMode?: BlendMode
       spread?: number
       visible?: boolean
+      showShadowBehindNode?: boolean
     }
 
+    export interface InnerShadowEffect {
+      type: 'inner-shadow'
+      color: HexCode | Color
+      offset: Vector
+      blur: number
+      blendMode?: BlendMode
+      spread?: number
+      visible?: boolean
+    }
+
+    export type ShadowEffect = DropShadowEffect | InnerShadowEffect
```

Comment:
This will keep it backward compatible right if people are already importing ShadowEffect?

Reply:
I think this is fine!